vue-vtkjs-pvw-template/src/store/busy.js

```diff
@@ -1,6 +1,4 @@
 /* eslint-disable no-unused-vars */
-import { Mutations } from 'vue-vtkjs-pvw-template/src/store/TYPES';
```

Comment:
That full file could disappear and be part of the wslink store since the wsLink client keep track of the busy part.